test/files/Elm.Native.Utils.erl

```diff
@@ -0,0 +1,8 @@
+-module('Elm.Native.Utils').
```

Comment:
`Elm.Native.*` should not be implemented here, I'm thinking of having a new repo on the elmer-compiler organization that mirror those from elm-lang, for example one repo for core, one for html, etc, and they *MUST* use the same semantic version than the package from elm. So for example, `Elm.Native.Utils` should be on `elmer-core` etc. IIRC the elm-compiler just concatenates those Native javascript modules when everything is compiled to javascript. But we must require the user to list each elmer-* native package as another erlang dependency at runtime.